src/app/common/themable-app.directive.ts

```diff
@@ -0,0 +1,82 @@
+import { DOCUMENT } from '@angular/common';
+import { Directive, HostListener, Inject, OnInit } from '@angular/core';
+
+const PREBUILT_THEME_MAP = {
+    'igx-theme': 'material-light',
+    'igx-dark-theme': 'material-dark',
```

Comment:
```suggestion
    'igx-material-light': 'material-light',
    'igx-material-dark': 'material-dark',
```

I believe all themes are of equal importance and we should material in the name for that reason.